src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/WorkloadSet.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.DotNet.MSBuildSdkResolver;
+using Strings = Microsoft.NET.Sdk.Localization.Strings;
+
+
+#if USE_SYSTEM_TEXT_JSON
+using System.Text.Json;
+#else
+using Newtonsoft.Json;
+#endif
+
+
+namespace Microsoft.NET.Sdk.WorkloadManifestReader
+{
+    public class WorkloadSet
+    {
+        public Dictionary<ManifestId, (ManifestVersion Version, SdkFeatureBand FeatureBand)> ManifestVersions = new Dictionary<ManifestId, (ManifestVersion version, SdkFeatureBand featureBand)>();
+
+        //  TODO: Generate version from hash of manifest versions if not otherwise set
```

Comment:
Is this TODO still needed? It looks like currently it's never being used and only added based on the manifest folder name.

Reply:
I expect it to be used in the future.  We will want to report the current workload set version from `dotnet workload --version`, and show the old and new workload set versions when updating workloads.